test/acceptance/features/steps/kind.py

```diff
@@ -91,28 +92,26 @@ def ipaddress(self):
 
 
 class PrimazaKind(PrimazaCluster):
+    __controller_image: str = get_env("PRIMAZA_CONTROLLER_IMAGE_REF")
+    __agentapp_image: str = get_env("PRIMAZA_AGENTAPP_IMAGE_REF")
+    __agentsvc_image: str = get_env("PRIMAZA_AGENTSVC_IMAGE_REF")
+
```

Comment:
error handling? what if the env var is not set? Maybe set defaults in the get_env call here as well.

Reply:
OK